### ISSUE TITLE
translate certain types of nodes or create new ones?

### ISSUE THREAD

**Issue by mikefantini** [DoSomethingArchive/legacy-website#5022]
this kind of makes my head hurt and is maybe overthinking it on some of these but here we go..

types of nodes that i have questions on (ranked in order of what I think matters):
- homepage
- not found
- facts
- images

homepage:

per #4968:
- I think it makes most sense to give Brazil Admin, Mexico Admin, and Global Admin access to create a home node and then just serve up the appropriate homepage based on where the user is coming from.
- The other option would be to make all of the pages on the home node translatable, including which of the 9 campaigns we're featuring on the homepage.
- I say we just go with whatever is easiest to implement.

not found:
- I'm just not sure if translating the existing Not Found nodes would make it easier or harder to display 404 content in Mexican Spanish to Mexican users. 
- I think it would make it easier, because right now the 404 page just randomly selects a published Not Found node and displays that to the user. Translating these I think would simplify logic.

facts:
- I say create new ones rather than translate the node, because even if the fact node is translated, anywhere we use it, we'd have to translate the rest of the content on the page anyway and the benefit seems pretty small of having the existing fact translated rather than just creating a new fact in Mexican Spanish and using that. 
- Also if we translate them, then it may cause some funkiness on fact pages where some of the facts show up in Mexican Spanish and some in US English if that fact node is translated and exists on both a campaign node and fact page node.

images:
- it would be nice to translate images so that even if we can't get the copy for a particular node in the user's copy, the image could be relevant to those users (especially since imagery is so huge on our site)
- however one issue I could see if this getting messed up with translators forgetting one crop or another and the image not displaying properly to users
- probably not worth translating and more trouble than it's worth

related to #4968 and #5020 

if we decide that we should translate instead of create new ones, then BR/MX admins would need access to:
- `Fact: Edit any content`
- `Image: Edit any content`
- `Not Found: Edit any content`

cc: @julielorch @angaither @mshmsh5000 @blisteringherb @sheydari-ds

**Comment by blisteringherb** [comment]
@mikefantini In terms of facts, on the one hand I'm thinking about translated facts as localized versions of the same fact which would then mean they were grouped in a logical way together, but OTOH, I think we'd need to do some more discovery to figure out how node references work with translations.  I'd love it if at some point we could continue to streamline translating campaigns and even maybe fact pages by displaying translated facts if they're available, but that's more work.

tl;dr I agree with your assessment here. For images maybe we can eventually put in some logic to prevent a translation from being saved if a crop isn't present.

**Comment by mikefantini** [comment]
thanks @blisteringherb do you have any thoughts on homepage?

@angaither @mshmsh5000 would love if either of you have thoughts on these too.

**Comment by mikefantini** [comment]
we decided that any nodes that are referenced on other nodes and not accessed by URL individually we would not translate and instead would just reference a new node in the appropriate language.

so we _won't_ be translating facts or images.

we will need to translate homepage (v1.0) and not found (v1.1).

one potential downside (?) to this is that facts will be in multiple languages and if we ever decide to translate those nodes changing the source language per fact could get hairy.
